### PR TITLE
change from CircleCI to TravisCI

### PR DESCRIPTION
**Pull request by rjw57**
We have a simple set up and have no need for the private repo support.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+dist: trusty
+sudo: false
+language: python
+python:
+    - "3.4"
```
Comment (abrahammartin) [inline .travis.yml:5]: shouldn't we test against 3.6? We use 3.6 in our dockerfile, no?

**Review by abrahammartin — COMMENTED** [review]
looks good except for a minor comment

File: .travis.yml (hunk 1 above)
Comment (rjw57) [reply .travis.yml:5]: Yes, this was a hang over from when we were using the standard GCE image. Will change.

**Comment by rjw57** [comment]
Huh - Apparently I changed my mind on Travis FSR since I closed this PR and deleted the branch. I don't recall why now :(. Perhaps it was an attempt to get the coveralls bot to work?

**Pull request merged**